src/dispatch/static/dispatch/src/signal/TableInstance.vue

```diff
@@ -30,6 +30,21 @@
                 {{ item.project.name }}
               </v-chip>
             </template>
+            <template v-slot:item.filter_action="{ item }">
+              <v-chip
+                small
+                text-color="white"
+                :color="
+                  item.filter_action === 'snooze'
+                    ? 'red'
```

Comment:
Hmm, I wonder if we should pick something other than red? It's not anything bad and red is our primary color.

Reply:
Yeah I wasn't sure what color to use, I'll do blue / light-blue.